### ISSUE TITLE
Plugin maintenance?

### ISSUE THREAD

**Issue by thepuzzlemaster** [PostHog/posthog-filter-out-plugin#9]
Hey, I'm curious if this plugin is actively maintained. I'm potentially interested in extending its functionality and I'm unclear if doing that here or creating a new plugin is going to be the way to go. I'd like to make the filtering syntax a bit more flexible, so that we can only filter certain events based on criteria. It would necessitate extending the syntax. But given some of the design decisions that were made with this plugin, I'm not sure if extending the existing interface without creating breaking changes is going to be a worse idea than just creating a new plugin entirely.

I'm also not familiar with the plugin ecosystem for Posthog, so I'm not sure how actively these are maintained, and if anyone has any real opinion on direction here.

**Comment by JamesJosephFinn** [comment]
I agree that this plugin is a necessary extension to Posthog; and that its current implementation is rather confusing UX-wise.
